src/ITensorsExtensions/ITensorsExtensions.jl

```diff
@@ -52,16 +54,26 @@ invsqrt_diag(it::ITensor) = map_diag(inv ∘ sqrt, it)
 pinv_diag(it::ITensor) = map_diag(pinv, it)
 pinvsqrt_diag(it::ITensor) = map_diag(pinv ∘ sqrt, it)
 
-function map_itensor(
-  f::Function, A::ITensor, lind=first(inds(A)); regularization=nothing, kwargs...
+function map_eigenvalues(
```

Comment:
Let's call this `map_eigvals` since Julia's `LinearAlgebra` library uses `eigvals(...)` to get the eigenvalues of a matrix.

Reply:
Sure thing.